src/protocols/secure_channel/CASESession.cpp

```diff
@@ -91,26 +89,15 @@ enum CASETLVTag : uint8_t
     kNOC = 6,
     /*! \brief Tag 7. The packet contains a signature. */
     kSignature = 7,
-    /*! \brief Tag 8. The packet contains a Trusted Root ID. */
-    kTrustedRootID = 8,
-    /*! \brief Tag 9. The packet contains an Encrypted data blob. */
-    kEncryptedData = 9,
-    /*! \brief Tag 10. The packet contains an AEAD Tag. */
-    kTag = 10,
-    // TODO: Remove tag 11
-    /*! \brief Tag 11. The packet contains the total number of Trusted Root IDs. */
-    kNumberofTrustedRootIDs = 11,
+    /*! \brief Tag 8. The packet contains an Encrypted data blob. */
+    kEncryptedData = 8,
+    /*! \brief Tag 9. The packet contains an AEAD Tag. */
+    kTag = 9,
```

Comment:
This entire enum "shifted" from original meaning to being TLV tags. The Tag IDs are actually in the spec. These tag IDs are all not to-spec:

```
[[ref_Sigma2]]
====== Generate and Send Sigma2

If <<ref_VALIDATE_S1, validation>> is successful, the responder encodes and sends a `Sigma2` message.

[[ref_sigma-2-struct]]
[subs="+macros"]
----
sigma-2-tbsdata => STRUCTURE [ tag-order ]
{
  responderNOC       [1] : OCTET STRING,
  responderEphPubKey [2] : <<ref_ec-pub-key, ec-pub-key>>,
  initiatorEphPubKey [3] : <<ref_ec-pub-key, ec-pub-key>>,
}

sigma-2-tbedata => STRUCTURE [ tag-order ]
{
  responderNOC [1] : OCTET STRING,
  signature    [2] : <<ref_ec-signature, ec-signature>>,
}

sigma-2-struct => STRUCTURE [ tag-order ]
{
  responderRandom    [1] : OCTET STRING [ length 32 ],
  responderSessionId [2] : UNSIGNED INTEGER [ range 16bits ],
  responderEphPubKey [3] : <<ref_ec-pub-key, ec-pub-key>>,
  encrypted2         [4] : OCTET STRING,
}
```

^^^ Tag IDs are contextual to a structure.

Reply:
Thanks, updated.